config/locales/en/documents/history.yml

```diff
@@ -24,8 +25,11 @@ en:
         draft_reset: "Draft reset to last published edition"
         unwithdrawn: "Unwithdrawn"
         scheduled: "Scheduled to publish"
+        scheduled_publishing_succeeded: "Published on schedule"
```

Comment:
Looks like we've got a duplicate here; the same translation is on line 11 already.

Reply:
Oops - thanks for catching that!